docs/changelog.md

```diff
@@ -2,6 +2,18 @@
 
 ## CARET
 
+### v0.4.10 <small>\_ Jun 12, 2023</small> {id = "0.4.10"}
+
+- **Update**: Changed trace points used to tie records of inter-process communication. This change affects trace data compatibility. However, this only affects cases where trace data captured after this update is analyzed by caret_analyze before this update. In that case, update caret_analyze ([CARET_analyze #296](https://github.com/tier4/CARET_analyze/pull/296)), ([CARET_trace #113](https://github.com/tier4/CARET_trace/pull/113)), ([CARET_doc #191](https://github.com/tier4/CARET_doc/pull/191)), ([ros2caret #70](https://github.com/tier4/ros2caret/pull/70))
+  - Please see the table below for the supported combinations
+
+|                               | trace data (<=0.4.9) | trace data (>=v0.4.10) |
+| ----------------------------- | :------------------: | :--------------------: |
+| **CARET_analyze (<=v0.4.9)**  |          OK          |           NG           |
```

Comment:
Plain text is fine, but "NG" is not a normal English. Why don't you use "Supported" / "Unsupported"? (or  "Compatible" / "Incompatible")

Reply:
I fixed the words at `5c25d91`.